_rules/SC4-1-2-aria-attr-valid.md

```diff
@@ -0,0 +1,118 @@
+---
+name: Validity of ARIA attribute
+
+description: |
+ This rule checks that the `aria-` attribute specified is defined in ARIA 1.1.
+
+success_criterion:
+- 4.1.2
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+Any attribute that starts with `aria-`.
+
+### Expectation
+
```

Comment:
Does this rule only take into account if the attribute exists somewhere in the ARIA spec, or does it also consider whether it is a supported state/property for that role? 
The variation in elements in the test cases seem to indicate the last, but it is not specified in the Expectations.

Reply:
I think "starts with 'aria-'" is bad practice. It should be "Any attribute defined by <ARIA-spec-location>" or such.

Reply:
In this rule the applicability needs to be "starts with aria-" because the rule is checking if anything that starts with aria- is indeed in the ARIA specification.

Reply:
I agree with @annethyme that we need to make it clear if the rule just checks if the attribute exists in the spec, or if it looks also at it being supported for the role (which I don't think if the case for this rule). A note would make it explicit.

Reply:
@annethyme - the rule only takes into account that the attribute exists in the ARIA spec.

@nitedog - The currently applicability and expectations is very simple and straight-forward. Your suggestion `any attribute defined by ...` will combine the expectation with the applicability. I would prefer to leave it as is. 

@carlosapaduarte - hope the above two comments answers your concerns too.